EIPS/eip-100.md

```diff
@@ -35,5 +35,5 @@ Changing the denominator from 10 to 9 ensures that the block time remains roughl
 
 ### References
 
-1. EIP 100 issue and discussion: https://github.com/ethereum/EIPs/issues/100
+1. EIP 100 issue and discussion: https://eips.ethereum.org/EIPS/eip-100
```

Comment:
This seems to be a recursive declaration.

Reply:
Maybe the solution here is that these self-links can go under the discussion-to metadata and I can take them out of the references like this.

(But if there is a specific comment that is being linked to I would keep that.)

What do you think about that?

Reply:
I'm a bit confused, why is the original reference to the discussion forum a bad thing?

Reply:
For most situations I think it makes sense to link to the web page (i.e. eips.ethereum.org/eip-number) but in this case where we're referencing the "issue and discussion" I feel like the GitHub link is most relevant, so that people can view & add to the discussion.

Reply:
To clarify, this change is in the EIP-100 file, which is rendered on eips.ethereum.org/EIPS/eip-100 where it links back to itself. If that reference link is kept, it makes a lot of sense to keep it as a useful link.